### ISSUE TITLE
Drop `is_webextension` field from `files` table in database

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#18409]
We removed `is_webextension` from the model (https://github.com/mozilla/addons-server/pull/18018) but it looks like we haven't dropped it from the database table yet. It should just be a matter of generating the migration.

**Comment by tejaram15** [comment]
@diox I have created the pull request for the above change. Please review and let me know if any changes have to be made.

**Comment by diox** [comment]
No, serializers are used for our API and `is_webextension` should be kept there for backwards-compatibility purposes.
